.github/workflows/prerelease.yml

```diff
@@ -5,36 +5,44 @@ on:
     branches:
       - main
 
+permissions:
+  contents: write
+
 env:
   OHAI_PFX_CERTIFICATE_BASE64: ${{ secrets.OHAI_PFX_CERTIFICATE_BASE64 }} # base64 encoded
   OHAI_PFX_PASSPHRASE:  ${{ secrets.OHAI_PFX_PASSPHRASE }}
-  GO_VERSION: '1.18'
 
 ###
 ###   PRE-RELEASE CREATION # Creates a pre-release if needed and loads variables for next jobs
 ###
 name: Create Pre-release
 jobs:
-  create_prerelease:
-    name: Create Pre-release
+  exporter_variables:
+    name: Test exporter e2e and output exporter variables
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/setup-go@v2
-        with:
-          go-version: ${{env.GO_VERSION}}
-      - name: Checkout code
-        uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           fetch-depth: 0
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: nri-config-generator/go.mod
+
       - name: Check if exporters have been modified
         id: check
         # Running shouldDoRelease we check if any exporter.yml has been modified, if so we set CREATE_RELEASE to true setting the variable EXPORTER_PATH
-        # Tha variable is used by the following step and jobs
+        # That variable is used by the following step and jobs
         run : |
           source ./scripts/common_functions.sh
           shouldDoRelease
```

Comment:
Or even releasing all of them when anything changes in `nri-config-generator`... Currently the release is onlytriggered when the yaml file is changes because it seems to be defined as the source of truth for the exporter package. We could revisit this or keep as it its but it is definitely out of the scope of this PR.